.nengobones.yml

```diff
@@ -239,6 +239,9 @@ travis_yml:
 
 ci_scripts:
   - template: static
+    codespell_ignore_words:
+      - nin
+      - nIn
```

Comment:
I think we don't need to ignore these words any more right, after the changes here (or in the deterministic-plan PR)?

Reply:
Correct, I don't think we need these ignores anymore.